docs/assistant/connect-to-bedrock.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[assistant-connect-to-bedrock]]
+= Connect to Amazon Bedrock
+
+This page provides step-by-step instructions for setting up an Amazon Bedrock connector for the first time. This connector type enables you to leverage large language models (LLMs) within {kib}. You'll first need to configure AWS, then configure the connector in {kib}.
+
+NOTE: Only Amazon Bedrock's `Anthropic` models are supported: `Claude` and `Claude instant`.
+
+[discrete]
+== Configure AWS
+
+[discrete]
+=== Configure an IAM policy 
+
+. Log into the AWS console and search for Identity and Access Management (IAM).
+. From the **IAM** menu, select **Policies** → **Create policy**.
+. To provide the necessary permissions, paste the following JSON into the **Specify permissions** menu.
++
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "bedrock:InvokeModel",
+                "bedrock:InvokeModelWithResponseStream"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
++
+NOTE: These are the minimum required permissions. IAM policies with additional permissions are also supported.
++
+. Click **Next**. Name your policy.
+
+[discrete]
+=== Configure an IAM User
+
+. Return to the **IAM** menu. Select **Users** from the navigation menu, then click **Create User**. 
+. Name the user, then click **Next**.
+. Select **Attach policies directly**. 
+. In the **Permissions policies** field, search for the policy you created earlier, select it, and click **Next**.
+. Review the configuration then click **Create user**.
+
+[discrete]
+=== Create an access key
+
+. Return to the **IAM** menu. Select **Users** from the navigation menu. 
+. Search for the user you just created, and click its name.
+. Go to the **Security credentials** tab.
+. Under **Access keys**, click **Create access key**.
+. Select **Third-party service**, check the box under **Confirmation**, click **Next**, then click **Create access key**.
+. Click **Download .csv file** to download the key. Store it securely.
+
+[discrete]
+=== Enable model access
+
+. Search the AWS console for Amazon Bedrock. 
+. From the Amazon Bedrock page, click **Get started**.
+. Select **Model access** from the left navigation menu, then click **Manage model access**.
+. Check the boxes for **Claude** and **Claude Instant**, depending which model or models you plan to use.
```

Comment:
Check the boxes for **Claude** and/or **Claude Instant**, depending which model or models you plan to use.